manifests/config.pp

```diff
@@ -48,9 +48,30 @@
       mode   => '0644'
     }
 
+    file { $elasticsearch::params::logdir:
+      ensure  => 'directory',
+      mode    => '0644',
+      recurse => true
+    }
+
+    file { $elasticsearch::params::homedir:
```

Comment:
We discussed this over video; it'll change bin/elasticsearch's permissions. Richard's working on a fix!